tests/bin/check_tron_jobs_test.py

```diff
@@ -4,13 +4,32 @@
 import time
 
 import check_tron_jobs
+import mock
 from check_tron_jobs import State
+from mock import PropertyMock
 from testify import assert_equal
 from testify import TestCase
 
 
 class CheckJobsTestCase(TestCase):
 
+    def test_check_job_result_exception(self):
+        with mock.patch('check_tron_jobs.parse_cli') as mock_args, \
```

Comment:
This isn't very pretty. Can you use the decorator method instead? I think it helps reduce the nestedness.